<launch>
    <!-- Load the PCL nodelet manager -->
    <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

    <!-- Voxel Grid Filter Nodelet -->
    <node pkg="nodelet" type="nodelet" name="voxel_filter" args="load pcl/VoxelGrid nodelet_manager">
        <param name="input" value="/cropped_pointcloud" />
        <param name="output" value="/filtered_point_cloud" />
        <param name="leaf_size" value="0.1" />
    </node>
</launch>
